### PR TITLE
Updating fromSecret function description

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/14744

### CODE REVIEW

File: governance/custom_template.adoc [hunk 1]
```diff
@@ -38,8 +38,9 @@ The `fromSecret` function returns the value of the given data key in the secret.
 ----
 func fromSecret (ns string, secretName string, datakey string) (dataValue string, err error)
 ----
-
-When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the secret or the data key does not exist on the target cluster. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
+//or should i replace, "You receive a policy violation if the data key does not exist on the target cluster."
+//with, "If the data key does not exist on the target cluster, the value becomes an empty string."
+When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the data key does not exist on the target cluster. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
```
Comment (mprahl) [inline governance/custom_template.adoc]:
```suggestion
When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the secret does not exist on the target cluster. If the data key does not exist on the target cluster, this function will return an empty string. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
```

**Comment by dockerymick** [comment]
@mprahl Thanks for the suggestion. I have updated the file, may you rereview this PR please?

File: governance/custom_template.adoc [hunk 2]
```diff
@@ -39,7 +39,7 @@ The `fromSecret` function returns the value of the given data key in the secret.
 func fromSecret (ns string, secretName string, datakey string) (dataValue string, err error)
 ----
 
-When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the secret or the data key does not exist on the target cluster. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
+When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the data key does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
```
Comment (mprahl) [inline governance/custom_template.adoc]:
```suggestion
When you use this function, enter the namespace, name, and data key of a Kubernetes `Secret` resource. You receive a policy violation if the Kubernetes `Secret` resource does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a `Secret` resource on the target cluster. The value for the `PASSWORD` data key is a template that references the secret on the target cluster:
```

File: governance/custom_template.adoc [hunk 3]
```diff
@@ -79,7 +79,7 @@ The `fromConfigmap` function returns the value of the given data key in the Conf
...
+When you use this function, enter the namespace, name, and data key of a Kubernetes resource. You receive a policy violation if the data key does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a Kubernetes resource on the target managed cluster. The value for the `log-file` data key is a template that retrieves the value of the `log-file` from the ConfigMap, `logs-config` from the `default` namespace, and the `log-level` is set to the data key `log-level`. If the data key does not exist in the ConfigMap or if the value is empty, then the value defaults to `ERROR`:
```
Comment (mprahl) [inline governance/custom_template.adoc]:
```suggestion
When you use this function, enter the namespace, name, and data key of a Kubernetes `ConfigMap` resource. You receive a policy violation if the Kubernetes `ConfigMap` resource does not exist on the target cluster. If the data key does not exist on the target cluster, the value becomes an empty string. View the following configuration policy that enforces a Kubernetes resource on the target managed cluster. The value for the `log-file` data key is a template that retrieves the value of the `log-file` from the ConfigMap, `logs-config` from the `default` namespace, and the `log-level` is set to the data key `log-level`.
```

**Comment by dockerymick** [comment]
Since I have accepted the suggestions, I am comfortable with merging this PR. If there are other changes that you notice in the file. Please comment on the issue

**Pull request merged**